### DISCUSSION TITLE
Is it somehow possible to localize that text values in Search Filters web part?

### DISCUSSION THREAD

**Discussion by AtomTS19** [microsoft-search/pnp-modern-search#2384] [Q&A]
![image](https://user-images.githubusercontent.com/70215912/191923832-16af5606-cc21-4d3a-8310-da2b6899ecf5.png)

**Comment by wobba** [comment]
Should be localized already to the supported UX languages. English, French, German, Spanish, Norwegian, Swedish, Danish, Finnish, Dutch and Polish.

**Comment by AtomTS19** [comment]
Hi Mikael,

but I need Czech language, so some unsupported. Just need to know if there
is some option or not.

thank you
regards
Tomas Smid

**Reply by wobba** [reply]
If you find someone to translate to Czech and do a PR with it we will include it.

**Comment by kasperbolarsen** [comment]
Czech has been one of the supported languages since v4.17ish
